### DISCUSSION TITLE
Emails in quarantine are not deleted.

### DISCUSSION THREAD

**Discussion by thelittlerocket** [simple-login/app#755] [Self-hosting question] [answered]
Hi there,

Thanks for this great app, it is awesome.
Today I found that the emails have been quarantined never getting deleted.
Would like to know if it is safe to delete those emails manually?
The oldest one is more than one year ago.

postgres:12.1
simplelogin/app:3.3.0

Any help is appreciated.
Thanks,

**Comment by nguyenkims** [comment, marked as answer]
Yes it's safe to delete them. They are there just so you can view them. On simplelogin.io, those emails are deleted after 7 days.

**Reply by rohitsud** [reply]
Is there a way to have them deleted automatically after 7 days on a self-hosted instance?
